firmware/console/binary/output_channels.txt

```diff
@@ -13,28 +13,28 @@ bit sd_present
 bit sd_logging_internal
 bit triggerScopeReady
 bit antilagTriggered
-bit isFanOn;radiator fan
+bit isFanOn;Radiator Fan
 bit isO2HeaterOn;
 bit checkEngine;
 bit needBurn;
 
-bit sd_msd
-bit isFan2On
-bit acrActive;
-bit toothLogReady;
-bit isTpsError;
-bit isCltError;
-bit isMapError;
-bit isIatError;
-bit isTriggerError;
-bit hasCriticalError;
-bit isWarnNow;
-bit isPedalError;
-bit launchTriggered;
-bit isTps2Error;
-bit isIdleClosedLoop;
-bit injectorFault;
-bit ignitionFault;
+bit sd_msd;SD is MSD
+bit isFan2On;Radiator Fan2
+bit acrActive;Holley ACR Active
+bit toothLogReady;Tooth Logger Ready
+bit isTpsError;TPS Error
```

Comment:
How do we feel about something like `Error: TPS` so that these alphabetize better?

Reply:
Sounds good to me.  Same for "Warning: ...", "Ready: ...", "Active: ..." ("Triggered: ..." -- change to "Active"?), "Fault: ..." (change to "Error"?)?

Reply:
The other stuff doesn't seem to have many similar ones for different systems, let's just start with error flags for sensors and such